adafruit_sgp40.py

```diff
@@ -57,26 +69,52 @@ class SGP40:
             import busio
             import board
             import adafruit_sgp40
+            # If you have a temperature sensor, like the bme280, import that here as well
+            # import adafruit_bme280
 
         Once this is done you can define your `busio.I2C` object and define your sensor object
 
         .. code-block:: python
 
             i2c = busio.I2C(board.SCL, board.SDA)
             sgp = adafruit_sgp40.SGP40(i2c)
+            # And if you have a temp/humidity sensor, define the sensor here as well
+            # bme280 = adafruit_bme280.Adafruit_BME280_I2C(i2c)
 
-        Now you have access to the raw gas value using the :attr:`raw` attribute
+        Now you have access to the raw gas value using the :attr:`raw` attribute.
+        And with a temperature and humidity value, you can access the class function
+        :meth:`measure_raw` for a humidity compensated raw reading
 
         .. code-block:: python
 
             raw_gas_value = sgp.raw
+            # Lets quickly grab the humidity and temperature
+            # temperature = bme280.temperature
+            # humidity = bme280.relative_humidity
+            # compensated_raw_gas = sgp.measure_raw(
```

Comment:
I think we are missing 
```suggestion
            # compensated_raw_gas = sgp.measure_raw(temperature=temperature, relative_humidity=humidity)
```